### ISSUE TITLE
Wildcard namespace selector should also select elements without namespace

### ISSUE THREAD

**Issue by gotson** [jhy/jsoup#1565]
Hello,

given the following input:

```xml
<?xml version='1.0' encoding='utf-8'?>
<package xmlns="http://www.idpf.org/2007/opf" version="3.0" unique-identifier="uuid_id" prefix="calibre: https://calibre-ebook.com">
  <metadata xmlns:opf="http://www.idpf.org/2007/opf" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:dcterms="http://purl.org/dc/terms/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:calibre="http://calibre.kovidgoyal.net/2009/metadata">
    <opf:meta refines="#id" property="file-as">Incomplete Theft, The</opf:meta>
    <meta refines="#id-1" property="file-as">Burke, Ralph</meta>
  </metadata>
</package>
```

1. `metadata > meta[property=file-as]` will find `<meta refines="#id-1" property="file-as">Burke, Ralph</meta>`
2. `metadata > *|meta[property=file-as]` will find `<opf:meta refines="#id" property="file-as">Incomplete Theft, The</opf:meta>`

I would expect the 2nd selector to match both nodes.

Is that intended or is it a bug ?

It can be verified using https://try.jsoup.org/

Thanks

**Comment by jhy** [comment]
Hi -- agreed, I think it makes sense to consider this a bug, and make the wildcard support non-namespaced elements.

**Comment by jhy** [comment]
Thanks -- fixed in 1.141.
